Jenkinsfile

```
@@ -1,6 +1,6 @@
 #!/usr/bin/env groovy
 
-library("govuk@default-branch")
+library("govuk")
 
 node("mongodb-2.4") {
   govuk.buildProject(defaultBranch: "main")
```

```suggestion
  govuk.buildProject()
```

Oh this, can go now too!